packages/veritone-json-schemas/schemas/vtn-standard/object/object.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://docs.veritone.com/schemas/vtn-standard/object.json",
+  "title": "vtn-standard.object",
+  "description": "Standard engine output for Object at Veritone",
+  "type": "object",
+  "properties": {
+    "validationContracts": {
+      "type": "array",
+      "items": {
+        "const": "object"
+      }
+    },
+    "object": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "object": {
+            "$ref": "master.json#/definitions/objectResult"
```

Comment:
Are relative file references allowed?  And if so, shouldn't this be `"../master.json#/definitions/objectResult"`?